tools/whois.init

```diff
@@ -167,12 +167,20 @@ case "$1" in
                         exit 1
                 fi
                 ;;
+        disable)
+          PID=$(pgrep -f "$COMMAND_REGEXP")
+          echo -e "bean net.ripe.db.whois:name=Loadbalancer && run setLoadbalancerEnabled false" | java -jar $JMXTERMPATH --url $PID
+          ;;
+        enable)
```

Comment:
Isn't ```enable``` automatic at startup? Or should the healthcheck page wait for this JMX command before reporting OK?

Reply:
I think not marking as enable automatically at start up could come in handy. Eg during manual testing scenarios, where one need to check stuff, before signifying to F5 to include the node into the pool. 

The deployment script would be updated to include a call that enables after start up.

Reply:
OK